website/docs/r/globalaccelerator_listener.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: "aws"
+page_title: "AWS: aws_globalaccelerator_listener"
+sidebar_current: "docs-aws-resource-globalaccelerator-listener"
+description: |-
+  Provides a Global Accelerator listener.
+---
+
+# aws_globalaccelerator_listener
+
+Provides a Global Accelerator listener.
+
+## Example Usage
+
+```hcl
+resource "aws_globalaccelerator_accelerator" "example" {
+  name            = "Example"
+  ip_address_type = "IPV4"
+  enabled         = true
+
+  attributes {
+    flow_logs_enabled   = true
+    flow_logs_s3_bucket = "example-bucket"
+    flow_logs_s3_prefix = "flow-logs/"
+  }
+}
+
+resource "aws_globalaccelerator_listener" "example" {
+  accelerator_arn = "${aws_globalaccelerator_accelerator.example.id}"
+  client_affinity = "SOURCE_IP"
+  protocol        = "TCP"
+
+  port_range {
+    from_port = 80
+    to_port   = 80
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `accelerator_arn` - (Required) The Amazon Resource Name (ARN) of your accelerator.
+* `client_affinity` - (Optional) The value for the address type must be `IPV4`. Valid values are `NONE`, `SOURCE_IP`.
```

Comment:
Copypasta? 🍝 😄 We likely want to borrow some verbiage from https://docs.aws.amazon.com/global-accelerator/latest/api/API_CreateListener.html and link out to it for more information.